tools/reviewer/global_registry.prototxt

```diff
@@ -3,6 +3,13 @@ reviewer_config {
   display_name: "StartupOS"
   config_repo: "https://github.com/google/startup-os"
   webapp_url: "https://startupos-5f279.firebaseapp.com"
-  config_url: "https://firestore.googleapis.com/v1beta1/projects/startupos-5f279/databases/(default)/documents/reviewer/config_binary"
   default_port: 7000
+  firebase_config {
+    api_key: "AIzaSyAn-A7YXlqR2JNOvlbhkVwcnsOigmphQIw"
```

Comment:
Why do we have copy of firebase data here?

Reply:
That's the place it's stored, to allow the webapp to connect to Firestore.

Reply:
Oh... It might be not trivial goal to import data outside from webapp and also not in `.ts`

Reply:
It may require extra steps. as gulp

Reply:
As discussed, there's a job that periodically copies the data from the code to Firestore, so the webapp can access it from there.